client/store/productsReducer.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios'
+
+const LOAD_PRODUCTS = 'LOAD_PRODUCTS';
+
+const productReducers = (state = [], action) =>{
```

Comment:
I usually include comments, to remind myself of what each piece of code is doing - this for example is your reducer, comment could be:
// reducer function